### PR TITLE
Update big query script

### PR DESCRIPTION
**Pull request by LadyChristina**
All Submissions:

* [x] Have you followed the guidelines in our [Contributing documentation](https://blockchain-technology-lab.github.io/consensus-decentralization/contribute)?
* [x] Have you verified that there aren't any other open Pull Requests for the same update/change?
* [x] Does the Pull Request pass all tests?

# Description
Updated block data collection script so that it only fetches new blocks (that don't exist in the local files). Also only fetches blocks before a certain day (today by default). 
Also removed --force-query cmd arg as it seems unnecessary now.

### CODE REVIEW

File: data_collection_scripts/collect_block_data.py [hunk 1]
```diff
@@ -66,9 +84,12 @@ def collect_data(ledgers, force_query):
...
+        '--to_date',
+        type=hlp.valid_date,
+        default=datetime.today().strftime('%Y-%m-%d'),
+        help='The date until which to get data for (YYYY-MM-DD format). Defaults to today.'
     )
+
     args = parser.parse_args()
-    collect_data(ledgers=args.ledgers, force_query=args.force_query)
+    from_block = {ledger: get_last_block_collected(ledger) for ledger in args.ledgers}
+    collect_data(ledgers=args.ledgers, from_block=from_block, to_date=args.to_datey)
```
Comment (dimkarakostas) [inline data_collection_scripts/collect_block_data.py:95]: typo "to_datey"

File: data_collection_scripts/collect_block_data.py [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
     for ledger in ledgers:
         file = RAW_DATA_DIR / f'{ledger}_raw_data.json'
-        if not force_query and file.is_file():
-            logging.info(f'{ledger} data already exists locally. '
-                         f'For querying {ledger} anyway please run the script using the flag --force-query')
-            continue
         logging.info(f"Querying {ledger}..")
-        query = (queries[ledger])
+
+        query = (queries[ledger]).replace("{{block_number}}", str(from_block[ledger]) if from_block[ledger] else "-1").replace("{{timestamp}}", f"'{to_date}'")
```
Comment (dimkarakostas) [inline data_collection_scripts/collect_block_data.py:37]: I think this ```f"'{to_date}'"``` can just be ```to_date```.

**Review by dimkarakostas — COMMENTED** [review]
LGTM in general - fix the typo and merge

File: data_collection_scripts/collect_block_data.py (hunk 2 above)
Comment (LadyChristina) [reply data_collection_scripts/collect_block_data.py:37]: That's how I had it at first but there was an issue with the lack of quotation marks

Comment (dimkarakostas) [reply data_collection_scripts/collect_block_data.py:37]: What issue?

Comment (LadyChristina) [reply data_collection_scripts/collect_block_data.py:37]: image

**Pull request merged**